packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -23,6 +23,11 @@ import { VisuallyHidden } from "../VisuallyHidden"
 import { AutocompleteInputOption } from "./AutocompleteInputOption"
 import { AutocompleteInputOptionLabel } from "./AutocompleteInputOptionLabel"
 
+export interface FooterActions {
```

Comment:
When exporting anything, types included, be sure to prefix: `AutocompleteFooterActions`